great_expectations/checkpoint/checkpoint.py

```diff
@@ -170,7 +170,7 @@ def run(
         # and action_list are considered the "default" validation.
         using_default_validation = not self.validations and not validations
 
-        run_time = run_time or datetime.datetime.now()
+        run_time = run_time or datetime.datetime.now().astimezone()
```

Comment:
```suggestion
        run_time = run_time or datetime.datetime.now(tz=datetime.timezone.utc)
```
I think this is equivalent to what you have but with 1 less object creation.
https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow